### PR TITLE
fix tail plugin troubleshooting logs and tsg

### PR DESCRIPTION
**Pull request by NarineM**

### CODE REVIEW

File: source/code/plugins/tailfilereader.rb [hunk 1]
```diff
@@ -22,17 +22,17 @@ def initialize(paths)
...
     attr_reader :paths
 
     def file_exists(path)
       if File.exist?(path)
-        @log.info "Following tail of #{path}"
+        @log.info "File path #{path} exists. Trying to tail."
         return path
       else
-        @log.warn "#{path} does not exist. Cannot tail the file."
+        @log.info "#{path} does not exist or not accessable. Cannot tail the file. Skipping."
```
Comment (AaronRM) [inline source/code/plugins/tailfilereader.rb:35]: Spelling: accessible

Comment (AaronRM) [inline source/code/plugins/tailfilereader.rb:35]: Why is this (and other similar lines) changed from log.warn => log.info?

**Review by AaronRM — CHANGES_REQUESTED** [review]

File: docs/Troubleshooting.md [hunk 2]
```diff
@@ -424,10 +424,10 @@ sudo service crond start
 * Onboarding to OMS Service failed
 * The setting "Apply the following configuration to my Linux Servers" has not been check marked
 * omsconfig has not picked up the latest Custom Log from the portal
-* OMS Agent for Linux user `omsagent` is unable to access the Custom Log due to permissions or not being found
- * `[DATETIME] [warn]: file not found. Continuing without tailing it.`
- * `[DATETIME] [error]: file not accessible by omsagent.`
-* Known Issue with Race Condition fixed in OMS Agent for Linux version 1.1.0-217
+* OMS Agent for Linux user `omsagent` is unable to access the Custom Log due to permissions or path not being found. In that case osmagent.log will contain one of these messages:
+  * `[DATETIME] [info]: <path> not found. Continuing without tailing it.`
+  * `[DATETIME] [info]: <path> does not exist or not accessable. Cannot tail the file. Skipping.`
```
Comment (AaronRM) [inline docs/Troubleshooting.md:429]: Spelling: accessible

File: source/code/plugins/tailfilereader.rb (hunk 1 above)
Comment (NarineM) [reply source/code/plugins/tailfilereader.rb:35]: we want this message to be logged in default logs. Currently to enable debug logging , cx needs to make changes in their config files manually.

Comment (AaronRM) [reply source/code/plugins/tailfilereader.rb]: Info lines are included by default, but not warnings?

Comment (NarineM) [reply source/code/plugins/tailfilereader.rb]: yes

**Review by AaronRM — APPROVED** [review]

**Pull request merged**